topic04-git-introduction/book-git-tutorial/03.Rebasing-Without-Conflicts.md

```diff
@@ -0,0 +1,22 @@
+# *__Git rebase__*
+
+Push the changes by typing “git push <remote><branch>”, If the branch isn’t out of date, then the changes will push straight up to your github repo, If it is out of date however, you will have to rebase.
```

Comment:
The changes will still get pushed even if the branch is out of date. The problem occurs when you want to merge a branch that is out of date via a pr